homeassistant/helpers/entity.py

```diff
@@ -281,6 +261,32 @@ def async_set_context(self, context):
                             "https://goo.gl/Nvioub", self.entity_id,
                             type(self), end - start)
 
+        return (state, attr)
+
+    async def async_update_ha_state(self, force_refresh=False, stateattr=None):
```

Comment:
Do we want `stateattr` to be a part of the external interface? It seems like it would only be used internally.

Reply:
Fixed.